python/mleap/pyspark/feature/math_binary.py

```diff
@@ -89,6 +89,10 @@ def __init__(
         #
         # if operation is not None, we can proceed to instantiate the scala classes
         if operation:
+            if (inputA and defaultA is not None) or \
```

Comment:
It's better if we check the final product of params.
Because use can set params in `__init__`, `setParams`, and set it one by one.
And set it one by one is actually the most common use case.
```
binaryMathBinary().setOpertion(BinaryOperation.Multiply).setInputA("aa").setInputB("bb").setDefaultA(1).setDefaultB(2)
```